README.md

```diff
@@ -1 +1,73 @@
-# budget-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Budget App
+
+A Budget-app is mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+![screenshot](./screenshot.png)
+
+
+## Live Version
+
+[See Live Version](https://evening-caverns-47353.herokuapp.com/)
+[Demo Video](https://www.loom.com/share/4a087545c28044feb7ce7046d2ad37cd)
+## Built With
+
+- Ruby
+- Rails
+- Postgresql
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+Start by cloning the repository.
+
+`https://github.com/vickymarz/budget-app.git`
+
+Navigate to the created folder.
+
+`cd budget-app`
+
+### Prerequisites
+
+- A Command Line Interface.
+- Having Ruby installed.
+- Setup Rails
+
+### Install
+
+To install rails, run `gem install rails on your terminal`
+
+### Usage
+
+### Run tests
+
+To run tests enter `rspec` command on your terminal
+
+## Authors
+
+👤 **Okoroji Victor aka Vickymarz**
+
+- GitHub: [@githubhandle](https://github.com/vickymarz)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/okoroji-victor-ebube-8791741a0)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).